util/custom-response/src/main/java/com/brother/customresponse/controller/StudentController.java

```diff
@@ -0,0 +1,36 @@
+package com.brother.customresponse.controller;
+
+import com.brother.customresponse.payload.Response;
+import com.brother.customresponse.payload.StudentDto;
+import com.brother.customresponse.service.StudentService;
+import com.brother.customresponse.util.ResponseBuilder;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.Valid;
+
+@RestController
+public class StudentController {
```

Comment:
should add request mapping at the class level 
should follow universal API endpoint rules like
http://api.example.com/user-management/users
http://api.example.com/user-management/users/{id}